Gopkg.lock

```diff
@@ -365,6 +385,23 @@
   ]
   revision = "e19ae1496984b1c655b8044a65c0300a3c878dd3"
 
+[[projects]]
+  name = "google.golang.org/appengine"
```

Comment:
This just makes me want to cry.

Reply:
we could force everyone to use `sonobuoy gen` for gcp if you want...